queries/topNCountries.sql

```diff
@@ -1,34 +1,25 @@
-# The top N populated countries in the world where N is provided by the user.
-SELECT code, name, continent, region, population, (
-    SELECT name
-    FROM city ci
-    WHERE countrycode = co.code
-        AND ci.id = co.capital
-    ) AS capital
-FROM country co
+# The top N most populous countries in the world where N is provided by the user.
+SELECT country.code, country.name, continent, region, country.population, city.name AS capital
```

Comment:
inconsistency in the SELECT portion of the statement. i.e. "country.name, continent" would expect the same convesntion throughout.  My prefence would be tablename.coloumn but I'm not precious about it.

Reply:
the table.filedname is only necessary if there are fields with the same names in the tables that you're joining. example:
country has fields name, continent and population
city has fields name, district and population

when referring to `continent` you don't need the `country.` prefix as there is only one field called continent in the query, but when referring to `name` you have to specify whether it's the name in the city table or the name in the country table. if you don't specify that then you get SQL errors